libnacl/sealed.py

```diff
@@ -0,0 +1,33 @@
+import libnacl
+import libnacl.public
+import libnacl.dual
+
+class SealedBox(object):
```

Comment:
This should inherit from libnacl.base.BaseKey so it gets all the helper funcs for key retrieval and saving etc